### ISSUE TITLE
ps u locks inside container because of /proc/stat mounted via lxcfs

### ISSUE THREAD

**Issue by Jetman80** [lxc/lxcfs#442]
Hi!

I have several dozens of very same servers: Ubuntu Focal, lxd:
```
lxd     4.10      19009
```

And on one of them inside containers commands like `ps u` lock console infinitely, do not react on ctrl-c, cannot be killed. Only hard reset of the server helped.

This servers differs from the others in CPU (AMD EPYC 7502P 32-Core Processor).

I found https://github.com/lxc/lxcfs/issues/407 thread, tried to `umount -l /proc/stat` and `ps u` started working.

I have `lxcfs.cfs=true`, `lxcfs.loadavg=true`, `lxd lxcfs.pidfd=true` everywhere, now I am trying to disable it one by one to find out if it could help.

After reboot `ps u` works inside container for about 2 hours, then it stops. Approximately in the same time I have this in hosts log:
```
Jan 27 12:15:10 hetzner5 lxd.daemon[2440]: proc_cpuview.c: 815: cpuview_proc_stat: Write to cache was truncated
```

Perhaps lxcfs/lxd has some bug that appears only on systems with high cpu thread count? This cpu has 64 threads.

**Comment by brauner** [comment]
I'll take a look soon.

**Comment by brauner** [comment]
Ok, I think this is fixed by #443.
